rasa/core/events/__init__.py

```diff
@@ -33,8 +33,8 @@ def deserialise_events(serialized_events: List[Dict[Text, Any]]) -> List["Event"
                 deserialised.append(event)
             else:
                 logger.warning(
-                    f"Ignoring event ({event}) while deserialising "
-                    "events. Couldn't parse it."
+                    f"Unable to parse event '{event}' while deserialising. The event"
+                    " will be ignored."
```

Comment:
we should find a common style here. I think pycharm inserts `f` infront of every string part when hitting enter within a format string. I'd suggest we go for this everywhere, but not a strong opinion on this but should be the same.

Reply:
https://www.notion.so/rasa/Code-conventions-26835ffd72484dd699f28fa693916d2e